src/application/scene/Todo.java

```diff
@@ -1,14 +1,186 @@
 package application.scene;
 
 import javafx.fxml.FXML;
-import javafx.scene.control.Button;
+import javafx.scene.control.*;
+
+import java.io.*;
+import java.time.LocalDate;
+import java.util.List;
 
 public class Todo {
-	  @FXML private Button button_dashboard;
+    @FXML
+    private Button button_dashboard;
+    @FXML
+    private Button button_addEvent;
+    @FXML
+    private Button button_moveDOING;
+    @FXML
+    private Button button_moveDONE;
+    @FXML
+    private Button button_clearDONE;
+    @FXML
+    private TextField textField_eventName;
+    @FXML
+    private DatePicker datePicker;
+    @FXML
+    private ListView<String> listView_TODO;
+    @FXML
+    private ListView<String> listView_DOING;
+    @FXML
+    private ListView<String> listView_DONE;
+    @FXML
+    private ChoiceBox<String> choiceBox_status;
+
+
+    public void initialize() {
+
+        textField_eventName.setPromptText("TODO Events");
+
+        datePicker.setValue(LocalDate.now());
+
+        //Add TODO DOING DONE status to the choiceBox dropdown
```

Comment:
I think the code here is self explanatory so the comment could be removed

Reply:
Checked